src/content/terraform/product-landing.json

```diff
@@ -51,58 +51,56 @@
 	"blocks": [
 		{
 			"type": "heading",
-			"heading": "Use Cases",
+			"heading": "Featured Documentation",
 			"level": 2,
 			"size": 400
 		},
-		{
-			"type": "tutorial_cards",
-			"tutorialSlugs": ["vault/transform", "vault/static-secrets"]
-		},
 		{
 			"type": "linked_cards",
 			"cards": [
 				{
-					"heading": "Dolor sit amet",
-					"body": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incident this text is written so that it's very very long but it should be truncated in the card to a max of three lines to match designs",
-					"url": "https://www.terraform.io"
+					"heading": "Configuration Language",
+					"body": "Describe infrastructure in Terraform configuration language.",
+					"url": "/terraform/language"
 				},
 				{
-					"heading": "Dolor sit amet",
-					"body": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incident this text is written so that it's very very long but it should be truncated in the card to a max of three lines to match designs",
-					"url": "https://www.terraform.io"
+					"heading": "Terraform CLI",
+					"body": "Learn Terraform's CLI-based workflows.",
+					"url": "/terraform/cli"
 				},
 				{
-					"heading": "Dolor sit amet",
-					"body": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incident this text is written so that it's very very long but it should be truncated in the card to a max of three lines to match designs",
-					"url": "https://www.terraform.io"
+					"heading": "Terraform Cloud",
+					"body": "Collaborate with your team to provision infrastructure.",
```

Comment:
Do we have research on the most compelling pitch for TFC? Wondering how "collaborate with your team" stacks up against "Automated infrastructure provisioning using flexible workflows" or similar. (I'm trying to piece together a pitch by looking at: https://cloud.hashicorp.com/products/terraform)

Reply:
I don't think we have data on this. I pulled the descriptions from the docs page metadata and shortened them slightly so they didn't get automatically truncated

Reply:
Yeah this is something we can and should revisit at a later time -- good point alan! I have also come to understand that possibly collaboration features aren't THE most compelling draw of TFC at this stage. But for now, this is what matches the things we have everywhere else, so I say we keep it.